sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/sm_pipeline_with_comprehend.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<p>You could also mention how the customer can delete the Lambda function and IAM role created in this example, but that they don't incur costs if not deleted (if the Lambda isn't invoked). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>